packages/marketo-form/utils.ts

```diff
@@ -174,3 +174,56 @@ export function calculateDefaultValues(
   })
   return { ...initialValues, ...values }
 }
+
+export function segmentIdentify(leadFormFields: Record<string, unknown>) {
+  // This function is wrapped in a try/catch to prevent Segment errors from
+  // interrupting the form submission workflow.
+  try {
+    if (
+      typeof window !== 'undefined' &&
+      window.analytics &&
+      window.analytics.identify &&
+      typeof window.analytics.identify === 'function'
```

Comment:
There is a new analytics helper we can make use of here if you'd like https://github.com/hashicorp/web-platform-packages/blob/main/packages/analytics/analytics-js-helpers.ts#L7-L16

Reply:
ooooo

Reply:
added [`ca9c056` (#832)](https://github.com/hashicorp/react-components/pull/832/commits/ca9c056f4331698fbd738a59d53a46c7ef42baaf)